perf/CMakeLists.txt

```diff
@@ -5,6 +5,11 @@ add_subdirectory(lua)
 find_package(benchmark QUIET)
 if (NOT ${benchmark_FOUND})
     message(AUTHOR_WARNING "Google Benchmark library was not found")
+    set(MSG "Target test-c-perf is dummy, Google Benchmark library was not found")
+    add_custom_target(test-c-perf
+                      COMMAND ${CMAKE_COMMAND} -E cmake_echo_color --red ${MSG}
```

Comment:
Side note: Nicely done! Should be used widely in LuaJIT CMake machinery. Thanks!